openshift/e2e-tests-openshift.sh

```diff
@@ -26,71 +26,70 @@ readonly TARGET_IMAGE_PREFIX="$INTERNAL_REGISTRY/$EVENTING_NAMESPACE/knative-eve
+  oc new-project istio-operator
+  oc new-project istio-system
+  oc apply -n istio-operator -f https://raw.githubusercontent.com/Maistra/istio-operator/maistra-${MAISTRA_VERSION}/deploy/maistra-operator.yaml
 
   # Wait until the Operator pod is up and running
   wait_until_pods_running istio-operator || return 1
 
   # Deploy Istio
   cat <<EOF | oc apply -f -
-apiVersion: istio.openshift.com/v1alpha1
-kind: Installation
+apiVersion: istio.openshift.com/v1alpha3
+kind: ControlPlane
 metadata:
-  namespace: istio-operator
-  name: istio-installation
+  name: basic-install
 spec:
   istio:
-    authentication: false
-    community: true
+    global:
+      # use community images
+      hub: "maistra"
+      tag: ${MAISTRA_VERSION}.0
+      proxy:
+        resources:
+          requests:
+            cpu: 100m
+            memory: 128Mi
+          limits:
+            cpu: 200m
+            memory: 128Mi
+    sidecarInjectorWebhook:
+      enabled: false
+    gateways:
+      istio-egressgateway:
+        autoscaleEnabled: false
+      istio-ingressgateway:
+        autoscaleEnabled: false
+        ior_enabled: false
+    mixer:
+      policy:
+        autoscaleEnabled: false
+      telemetry:
+        autoscaleEnabled: false
+        resources:
+          requests:
+            cpu: 100m
+            memory: 1G
+          limits:
+            cpu: 200m
+            memory: 2G
+    pilot:
+      autoscaleEnabled: false
+    kiali:
+      enabled: false
+    tracing:
+      enabled: false
 EOF
 
-  # Wait until at least the istio installer job is running
-  wait_until_pods_running istio-system || return 1
+  # Wait until at least the Istio ControlPlane is installed
+  timeout 900 '[[ $(oc get ControlPlane/basic-install --template="{{range .status.conditions}}{{printf \"%s=%s, reason=%s, message=%s\n\n\" .type .status .reason .message}}{{end}}" | grep -c Installed=True) -eq 0 ]]' || return 1
 
-  timeout_non_zero 900 'oc get pods -n istio-system && [[ $(oc get pods -n istio-system | grep openshift-ansible-istio-installer | grep -c Completed) -eq 0 ]]' || return 1
-
-  # Scale down unused services deployed by the istio operator. The
-  # jaeger pods will fail anyway due to the elasticsearch pod failing
-  # due to "max virtual memory areas vm.max_map_count [65530] is too
-  # low, increase to at least [262144]" which could be mitigated on
-  # minishift with:
-  #  minishift ssh "echo 'echo vm.max_map_count = 262144 >/etc/sysctl.d/99-elasticsearch.conf' | sudo sh"
+  # Scale down unused services deployed by the istio operator
   oc scale -n istio-system --replicas=0 deployment/grafana
-  oc scale -n istio-system --replicas=0 deployment/jaeger-collector
-  oc scale -n istio-system --replicas=0 deployment/jaeger-query
-  oc scale -n istio-system --replicas=0 statefulset/elasticsearch
-
-  patch_istio_for_knative || return 1
```

Comment:
that is still used in serving:

https://github.com/openshift/knative-serving/blob/release-v0.6.0/openshift/e2e-tests-openshift.sh#L181

Reply:
I saw that, but it's only patching the sidecar config and we're not injecting sidecars with this change so it felt like dead code.